src/framework/dockwindow/internal/docktabbar.cpp

```diff
@@ -32,9 +32,11 @@ DockTabBar::DockTabBar(KDDockWidgets::TabWidget* parent)
 bool DockTabBar::event(QEvent* event)
 {
     switch (event->type()) {
-    //! NOTE: see https://github.com/musescore/MuseScore/issues/8164
-    case QEvent::MouseButtonDblClick:
+    case QEvent::MouseButtonDblClick: {
+        QMouseEvent* mouseEvent = static_cast<QMouseEvent*>(event);
+        doubleClicked(mouseEvent->pos());
```

Comment:
Without having looked into this very deeply yet, I'm wondering to what extent this overlaps with the functionality of `TabBarQuick::event` / `TabBar::onMouseDoubleClick` of KDDockWidgets itself.